pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -0,0 +1,562 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+// +build linux_bpf
+
+package tracer
+
+import (
+	"bufio"
+	"context"
+	"crypto/tls"
+	"fmt"
+	"golang.org/x/sys/unix"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"net"
+	nethttp "net/http"
+	"os"
+	"os/exec"
+	"regexp"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/network"
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	netlink "github.com/DataDog/datadog-agent/pkg/network/netlink/testutil"
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http"
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http/testutil"
+	"github.com/DataDog/datadog-agent/pkg/network/tracer/connection/kprobe"
+	"github.com/DataDog/datadog-agent/pkg/network/tracer/testutil/grpc"
+	"github.com/DataDog/datadog-agent/pkg/util/kernel"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func httpSupported(t *testing.T) bool {
+	currKernelVersion, err := kernel.HostVersion()
+	require.NoError(t, err)
+	return currKernelVersion >= http.MinimumKernelVersion
+}
+
+func httpsSupported(t *testing.T) bool {
+	return http.HTTPSSupported(testConfig())
+}
+
+func classificationSupported(config *config.Config) bool {
+	return kprobe.ClassificationSupported(config)
+}
+
+func TestEnableHTTPMonitoring(t *testing.T) {
+	if !httpSupported(t) {
+		t.Skip("HTTP monitoring not supported")
+	}
+
+	cfg := testConfig()
+	cfg.EnableHTTPMonitoring = true
+	tr, err := NewTracer(cfg)
+	require.NoError(t, err)
+	defer tr.Stop()
+}
+
+func TestHTTPStats(t *testing.T) {
```

Comment:
this test seems like it should be valid on windows?
Was it already linux only?
If already linux-only, it's fine, and we should (in a separate PR) move it over to windows/linux